### PR TITLE
Adding hexagon_map to documentation index for distro

### PR DESCRIPTION
**Pull request by DS921020**
I'd like my_repo hexagon_map to be indexed and documented on ros.org.

### CODE REVIEW

**Comment by DS921020** [comment]
I'd like my_repo hexagon_map to be indexed and documented on ros.org.

File: indigo/distribution.yaml [hunk 1]
```diff
@@ -4563,6 +4563,15 @@ repositories:
       url: https://github.com/heron/heron_desktop.git
       version: indigo-devel
     status: maintained
+  hexagon_map:
+     doc:
```
Comment (tfoote) [inline indigo/distribution.yaml]: This is failing CI due to being indented by 3 spaces not 2 here.

**Review by tfoote — CHANGES_REQUESTED** [review]
Please see the CI output. You need to make a small tweak to match the yaml syntax.

**Comment by tfoote** [comment]
Replaced by #18240

**Pull request closed without merging**